README.md

```diff
@@ -0,0 +1,76 @@
+# Reoserv.net documentation home
+
+This website hosts the documentation for the [reoserv][reoserv] project
+(authored and maintained by [sorokya][sorokya])
+
+## Prerequisites
+
+Before you can run this project, you need to have the following installed on
+your machine:
+
+- [Node.js][node] v20+ (for local development with the Remix web framework)
+- [Bun][bun] v1.0.36+ (a modern package manager & script runner for Node.js
+  projects)
+
+Verify that both are setup on your system and meet our version requirements by
+running:
+
+```sh
+bun --version    # eg: 1.0.36
+node --version   # eg: v20.4.0
+```
+
+Also, install the following VS Code extensions:
```

Comment:
I'd denote this as optional and maybe just link to the docs for the tools rather than the vscode marketplace